app/components/organization/SettingsMenu.js

```diff
@@ -101,6 +101,18 @@ class SettingsMenu extends React.Component {
         )
       },
       {
+        allowed: "teamAdmin",
```

Comment:
@ticky we have [a `see_account_audit_events` permission](https://github.com/buildkite/buildkite/blob/audit-ui-permission/app/models/permission.rb#L190-L192), maybe we need to [expose it](https://github.com/buildkite/buildkite/compare/master...audit-ui-permission) to use it here?